Content.Server/Salvage/SpawnSalvageMissionJob.cs

```diff
@@ -145,10 +153,13 @@ protected override async Task<bool> Process()
         expedition.EndTime = _timing.CurTime + mission.Duration;
         expedition.MissionParams = _missionParams;
 
+        // Assign coordinates to FTL destination
+        _xforms.SetCoordinates(FTLDestination, new EntityCoordinates(mapUid, grid.TileSizeHalfVector));
+
         // Don't want consoles to have the incorrect name until refreshed.
-        var ftlUid = _entManager.CreateEntityUninitialized("FTLPoint", new EntityCoordinates(mapUid, grid.TileSizeHalfVector));
-        _metaData.SetEntityName(ftlUid, SharedSalvageSystem.GetFTLName(_prototypeManager.Index<DatasetPrototype>("names_borer"), _missionParams.Seed));
-        _entManager.InitializeAndStartEntity(ftlUid);
+        _metaData.SetEntityName(FTLDestination, SharedSalvageSystem.GetFTLName(_prototypeManager.Index<DatasetPrototype>("names_borer"), _missionParams.Seed));
+        _entManager.GetComponent<FTLDestinationComponent>(FTLDestination).RequireCoordinateDisk = true;
+        _entManager.InitializeAndStartEntity(FTLDestination);
 
```

Comment:
FTLDestinationComponent goes on the map now so putting it on the beacon is unnecessary.